Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -462,7 +500,7 @@ final class CustomerCenterViewModelTests: TestCase {
                     entitlementId: "pro",
                     productId: productIdLifetime,
                     purchaseDate: purchaseDateLifetime,
-                    expirationDate: nil
+                    expirationDate: nil,
```

Comment:
I think this will break in older swift versions? I am unsure

Reply:
we need a formatter